### ISSUE TITLE
bug in coverage_plot?

### ISSUE THREAD

**Issue by sakoehler7** [sakoehler7/eesim#16]
It seems that when I run coverage_plot with any other true rr than 1.1 there are a lot more than 5 red bars on the plot.  I can't find what is causing this.

**Comment by geanders** [comment]
When I ran: 

```
ex_sim <- eesim(n_reps = 100, n = 10, central = 100, sd = 10,
          exposure_type = "continuous", average_outcome = 20, rr = 1.05,
          model = "spline", df_year = 1)
coverage_plot(ex_sim[[1]], true_param = 1.05)
```

a few times I got four and six red bars on the plot. 

However, when I tried using a lower `rr`, I did seem to get more than expected on average. I wonder if this is a convergence issue in the algorithm for very small relative risks? We can talk about this today and see if we can figure it out. I think it's an issue potentially with the simulation itself, rather than the plotting. I think it looks like this plotting function is working as expected.
